tests/lib/eZ/RichText/Converter/Xslt/_fixtures/docbook/031-ezstyle.xml

```diff
@@ -13,4 +13,11 @@
     <eztemplate name="highlighted_block" type="style" ezxhtml:align="center">
         <ezcontent>Highlighted block with "center" align.</ezcontent>
     </eztemplate>
+    <eztemplate name="highlighted_block" type="style">
+        <ezcontent>
+            Highlighted block with multiple lines
+            Highlighted block with multiple lines
```

Comment:
This doesn't seem like a correct XML. I'm not sure if `<br>` is proper for DocBook, but XML spec is rather not white-space-aware.